AndroidGesture/app/src/main/java/com/example/ben/androidgesture/PebbleReceiver.java

```diff
@@ -162,4 +170,29 @@ public void run() {
         }, 2000);
     }
 
+    public void onRadioButtonClicked(View view) {
```

Comment:
This function just needs to exist atm, cause im checking which radio is clicked in save data. I'm actually gonna change this right now